src/main/java/uk/gov/hmcts/reform/sandl.snlevents/mappers/FactsMapper.java

```diff
@@ -136,4 +138,12 @@ public String mapDbAvailabilityToRuleJsonMessage(Availability availability) thro
 
         return objectMapper.writeValueAsString(factAvailability);
     }
+
+    public String mapTimeToRuleJsonMessage(Time time) throws JsonProcessingException {
```

Comment:
Should be better name than Time, maybe Clock, or DateTimeSingleValue? Use random class name generator ;)